tests/Unit/ControlSystem/ControlErrors/Test_SizeError.cpp

```diff
@@ -203,7 +237,10 @@ void test_size_error_one_step(
     const auto control_error_history = error_class.control_error_history();
 
     // These should be identical because the control error class calls the
-    // control_error function
+    // control_error function. The horizon smoothing averager would normally
+    // make this slightly different, but it needs a few times before it can do
+    // anything. So in the meantime, the initial values for the horizon coef and
+    // its derivatives are the ones specified above
```

Comment:
I guess this means that the ah-smoothing-averager code is not tested, correct?  Probably should say this explicitly. Not sure how to easily test it (but testing it may be beyond the scope of this PR).